### PR TITLE
feat: add opentelemetry docs

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

File: guides/hosting/configurations/opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Enables the OpenTelemetry auto-instrumentation and sets the service name.
+
+```
+OTEL_PHP_AUTOLOAD_ENABLED=true
+OTEL_SERVICE_NAME=shopware
+```
+
+### Exporter configuration
+
+The OpenTelemetry extension needs to be configured to export the data to your collector. Here is an example todo that with with the OTLP exporter with GRPC:
```
Comment (keulinho) [inline guides/hosting/configurations/opentelemetry.md:46]: The second sentence is b0rked

File: guides/hosting/configurations/opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+```
+OTEL_TRACES_EXPORTER=otlp
```
Comment (keulinho) [inline guides/hosting/configurations/opentelemetry.md:49]: do i have other config options? seems weird as the option is already prefixed with OTEL :thinking: 

IMHO it should either be:
```
OTEL_TRACES_EXPORTER=1
```
or
```
TRACES_EXPORTER=otlp
TRACES_EXPORTER=blackfire
```

File: guides/hosting/configurations/opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+The OpenTelemetry instrucmentation collects following traces:
+
+- Controller
+- Symfony HTTP Client
+- MySQL Queries
+
+![Example Trace in Grafana](assets/otel-grafana-trace.png)
+
+### Adding custom spans
```
Comment (keulinho) [inline guides/hosting/configurations/opentelemetry.md:70]: This doesn't really belong here right? Shouldn't it be in a generic `Profiling` page and under that we have pages how to configure the concrete profiler that will be used?

Comment (keulinho) [reply guides/hosting/configurations/opentelemetry.md:70]: Or better:
Hosting
- Observability
  - Logging (monolog config)
  - Profiling / Tracing (how to add spans, how to configure blackfire, etc)
  - OpenTelemetry (how to use otel for logging & profiling)

File: guides/hosting/configurations/opentelemetry.md (hunk 2 above)
Comment (shyim) [reply guides/hosting/configurations/opentelemetry.md:49]: These are OTEL specific environment variables, we can't rename them 😅 
https://opentelemetry.io/docs/instrumentation/php/sdk/#configuration

Comment (keulinho) [reply guides/hosting/configurations/opentelemetry.md]: gotcha, then link it and describe that those envs come from the sdk and are not shopware specific

File: guides/hosting/configurations/observability/monolog.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
 ---
 nav:
-  title: Monolog
+  title: Logging
```
Comment (keulinho) [inline guides/hosting/configurations/observability/monolog.md:3]: maybe change the file name as well?

File: guides/hosting/configurations/observability/profiling.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    profiler:
+        integrations:
+            - Symfony
+            - Datadog
+            - Tideways
+            - OpenTelemetry
+```
+
+::: info
+The OpenTelemetry profiler is not installed by default. Checkout the [OpenTelemetry Integration](./opentelemetry.md) to learn how to install it.
```
Comment (keulinho) [inline guides/hosting/configurations/observability/profiling.md:27]: DD and tideways also need to have extensions installed or? Does it make sense to add a little section for each of these 

E.g. link to dd agent installation guide or tideways extension and so on

File: guides/hosting/configurations/observability/monolog.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
   position: 20
```
Comment (keulinho) [inline guides/hosting/configurations/observability/monolog.md:4]: everything has postition 20 :thinking: 
What order do we want to have? my suggestions would be:
* logging
* profiling
* OpeneTelemetry

going from the most basic to the most advanced

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+This file configures in production that the Shopware Profiler integration with OpenTelemetry is enabled and also configures that the Monolog output will be sent to OpenTelemetry.
+
+## Configuration
+
+After the installation, you will need to set some environment variables to configure the OpenTelemetry itself and exporter.
+
+### Basic configuration
+
+Enables the OpenTelemetry auto-instrumentation and sets the service name.
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:37]:
```suggestion
The following configuration enables the OpenTelemetry auto-instrumentation and sets the service name.
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+```
+OTEL_PHP_AUTOLOAD_ENABLED=true
+OTEL_SERVICE_NAME=shopware
+```
+
+You can find all possible environment variables [here](https://opentelemetry.io/docs/instrumentation/php/sdk/#configuration).
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:44]:
```suggestion
Refer to all possible [environment variables](https://opentelemetry.io/docs/instrumentation/php/sdk/#configuration) for better understanding.
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+OTEL_TRACES_EXPORTER=otlp
+OTEL_LOGS_EXPORTER=otlp
+OTEL_METRICS_EXPORTER=otlp
+OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
+```
+
+If you use gRPC with OTLP, you will need to install also `open-telemetry/transport-grpc open-telemetry/exporter-otlp` as composer packages.
+
+You can find more about the exporters [here](https://opentelemetry.io/docs/instrumentation/php/exporters/).
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:60]:
```suggestion
Refer to this docs for more information about the [exporters](https://opentelemetry.io/docs/instrumentation/php/exporters/).
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+- Controller
+- Symfony HTTP Client
+- MySQL Queries
+
+![Example Trace in Grafana](../../../../assets/otel-grafana-trace.png)
+
+## Example Grafana Stack
+
+You can find an example Stack [here](https://github.com/shopwareLabs/opentelemetry/tree/main/docker) with:
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:74]:
```suggestion
You can find an example [Stack](https://github.com/shopwareLabs/opentelemetry/tree/main/docker) with:
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+
+```
+OTEL_TRACES_EXPORTER=otlp
+OTEL_LOGS_EXPORTER=otlp
+OTEL_METRICS_EXPORTER=otlp
+OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
+```
+
+If you use gRPC with OTLP, you will need to install also `open-telemetry/transport-grpc open-telemetry/exporter-otlp` as composer packages.
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:58]: What does OTLP stand for ?

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
+---
+nav:
+  title: OpenTelemetry
+  position: 30
+
+---
+
+# OpenTelemetry
+
+OpenTelemetry is a standard to collect distributed traces, metrics and logs from the application. It's similar to tools like NewRelic, Datadog, Blackfire Monitoring and Tideways, but it's completely open source and vendor neutral. That means you can use OpenTelemetry to collect the data and push it to one of the vendors mentioned above, or you can use it to collect the data and push it to your own infrastructure with tools like Grafana Stack (Tempo, Loki, Prometheus, Grafana) or other tools.
```
[the remark this answers is not in the document]
Comment (shyim) [reply guides/hosting/configurations/observability/opentelemetry.md]: yes

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
+---
+nav:
+  title: OpenTelemetry
+  position: 30
+
+---
+
+# OpenTelemetry
+
+OpenTelemetry is a standard to collect distributed traces, metrics and logs from the application. It is similar to tools like NewRelic, Datadog, Blackfire Monitoring and Tideways, but it is completely open source and vendor neutral. That means you can use OpenTelemetry to collect the data and push it to one of the vendors mentioned above, or you can use it to collect the data and push it to your own infrastructure with tools like Grafana Stack (Tempo, Loki, Prometheus, Grafana) or other tools.
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:10]:
```suggestion
OpenTelemetry is a standard to collect distributed traces, metrics and logs from the application. It is similar to tools like NewRelic, Datadog, Blackfire Monitoring and Tideways, but it is completely open source and vendor neutral. That means you can use OpenTelemetry to collect the data and push it to one of the vendors mentioned earlier, or you can use it to collect the data and push it to your own infrastructure with tools like Grafana Stack (Tempo, Loki, Prometheus, Grafana) or other tools.
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+After the installation, you will need to set some environment variables to configure both, the OpenTelemetry and its exporter.
+
+### Basic configuration
+
+The following configuration enables the OpenTelemetry auto-instrumentation and sets the service name.
+
+```
+OTEL_PHP_AUTOLOAD_ENABLED=true
+OTEL_SERVICE_NAME=shopware
+```
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:42]:
```suggestion
```text
OTEL_PHP_AUTOLOAD_ENABLED=true
OTEL_SERVICE_NAME=shopware
```
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+
+The OpenTelemetry extension needs to be configured to export the data to your collector. Here is an example configuration for the OpenTelemetry Collector using gRPC:
+
+```
+OTEL_TRACES_EXPORTER=otlp
+OTEL_LOGS_EXPORTER=otlp
+OTEL_METRICS_EXPORTER=otlp
+OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
+```
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:56]:
```suggestion
```text
OTEL_TRACES_EXPORTER=otlp
OTEL_LOGS_EXPORTER=otlp
OTEL_METRICS_EXPORTER=otlp
OTEL_EXPORTER_OTLP_PROTOCOL=grpc
OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
```
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+
+```
+OTEL_PHP_AUTOLOAD_ENABLED=true
+OTEL_SERVICE_NAME=shopware
+OTEL_TRACES_EXPORTER=otlp
+OTEL_LOGS_EXPORTER=otlp
+OTEL_METRICS_EXPORTER=otlp
+OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
+```
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:92]:
```suggestion
```text
OTEL_PHP_AUTOLOAD_ENABLED=true
OTEL_SERVICE_NAME=shopware
OTEL_TRACES_EXPORTER=otlp
OTEL_LOGS_EXPORTER=otlp
OTEL_METRICS_EXPORTER=otlp
OTEL_EXPORTER_OTLP_PROTOCOL=grpc
OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
```
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

File: guides/hosting/configurations/observability/opentelemetry.md (hunk 11 above)
Comment (sushmangupta) [reply guides/hosting/configurations/observability/opentelemetry.md:56]: @shyim , just to confirm OTLP means OpenTelemetry Protocol ?

Comment (shyim) [reply guides/hosting/configurations/observability/opentelemetry.md:56]: Yes

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+```text
+OTEL_TRACES_EXPORTER=otlp
+OTEL_LOGS_EXPORTER=otlp
+OTEL_METRICS_EXPORTER=otlp
+OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
+
+If you use gRPC with OTLP, you will need to install also `open-telemetry/transport-grpc open-telemetry/exporter-otlp` as composer packages.
+
+Refer to this docs for more information about the [exporters](https://opentelemetry.io/docs/instrumentation/php/exporters/).
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:58]:
```suggestion
Refer to this doc for more information about the [exporters](https://opentelemetry.io/docs/instrumentation/php/exporters/).
```

File: guides/hosting/configurations/observability/opentelemetry.md [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+The OpenTelemetry extension needs to be configured to export the data to your collector. Here is an example configuration for the OpenTelemetry Collector using gRPC:
+
+```text
+OTEL_TRACES_EXPORTER=otlp
+OTEL_LOGS_EXPORTER=otlp
+OTEL_METRICS_EXPORTER=otlp
+OTEL_EXPORTER_OTLP_PROTOCOL=grpc
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
+
+If you use gRPC with OTLP, you will need to install also `open-telemetry/transport-grpc open-telemetry/exporter-otlp` as composer packages.
```
Comment (sushmangupta) [inline guides/hosting/configurations/observability/opentelemetry.md:56]:
```suggestion
If you use gRPC with OpenTelemetry Protocol (OTLP) , you will need to install `open-telemetry/transport-grpc open-telemetry/exporter-otlp` as composer packages.
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

**Review by sushmangupta — APPROVED** [review]
Thanks for the update! :)

File: .wordlist.txt [hunk 19]
```diff
@@ -1598,3 +1598,9 @@ YYYY
 zlib
 zsh
 ZSH
+Grafana
+OpenTelemetry
+OTLP
+gRPC
+Datadog
+NewRelic
```
Comment (sushmangupta) [inline .wordlist.txt:1606]:
```suggestion
NewRelic
OTEL
otel
otlp
grpc
```

**Pull request merged**